content/1_docs/3_reference/6_system/1_options/0_content/cheatsheet-article.txt

```diff
@@ -4,7 +4,7 @@ Excerpt: Set how Kirby handles content
 
 Text:
 
-## Content file extension
+## File extension
```

Comment:
I know we are in the context of content, but still I prefer these redundant title

Reply:
👍